src/primitives.test.ts

```diff
@@ -55,6 +56,102 @@ describe('primitives', () => {
     })
   })
 
+  describe('endsWith', () => {
```

Comment:
No real changes here, just moved because I accidentally added it out of alphabetical order. (Also pulled `'abc'` out to a variable)